tests/FakeItEasy.Tests/ACallToTests.cs

```diff
@@ -17,6 +20,21 @@ public ACallToTests()
             this.StubResolve(this.configurationManager);
         }
 
+        public static IEnumerable<object[]> CallSpecificationActions
+        {
+            get
+            {
+                var actions = new Action<IFoo>[]
+                {
+                    foo => A.CallTo(() => foo.Bar()),
+                    foo => A.CallTo(() => foo.Baz()),
+                    foo => A.CallToSet(() => foo.SomeProperty),
+                    foo => A.CallTo(foo),
+                };
+                return actions.Select(action => new object[] { action });
+            }

```

Comment:
I was surprised that you had to write all this.
If you're interested in shortening this, we could add a `TestCases.FromObject` overload:

```c#
public static IEnumerable<object[]> FromObject<T>(params T[] cases)
{
    return cases.Select(@case => new object[] { @case });
}
```